plugins/cri/cri.go

```diff
@@ -98,15 +120,16 @@ func initCRIService(ic *plugin.InitContext) (interface{}, error) {
 	}
 
 	options := &server.CRIServiceOptions{
+		RuntimeService:     criRuntimePlugin.(server.RuntimeService),
 		ImageService:       criImagePlugin.(server.ImageService),
 		NRI:                getNRIAPI(ic),
 		Client:             client,
 		SandboxControllers: sbControllers,
-		BaseOCISpecs:       criBase.BaseOCISpecs,
 	}
 	is := criImagePlugin.(imageService).GRPCService()
 
-	s, rs, err := server.NewCRIService(criBase.Config, options)
+	// TODO: More options specifically for grpc service?
```

Comment:
q: Out of curiosity. What are the more options for grpc service?

Reply:
I was mainly looking at the streaming configuration since I might consider that more owned by the API server and it could possibly make more sense to just pass the streaming server to the main server handler.